### PR TITLE
Ivanti.VelocityConsole version 2.1.16.21039

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107475)

### CODE REVIEW

File: manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ivanti.VelocityConsole
+PackageVersion: 2.1.16.21039
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/Velocity/velocity_console_2.1.16.msi
```
Comment (stephengillie) [inline manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.installer.yaml:7]:
```suggestion
- InstallerUrl: https://www.wavelink.com/download/Downloads.aspx?DownloadFile=27577&returnUrl=/Download-Velocity_enterprise-app-modernization-Software/
```

Comment (stephengillie) [reply manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.installer.yaml:7]: This package doesn't appear to be available from the publisher without registration. Packages generally have to be available for anonymous download from almost any terrestrial location at any time.

File: manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Ivanti.VelocityConsole
+PackageVersion: 2.1.16.21039
+PackageLocale: en-US
+Publisher: Ivanti
+PackageName: Velocity Console
```
Comment (stephengillie) [inline manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.ivanti.com/solutions/supply-chain?wlredirect
PublisherSupportUrl: https://www.ivanti.com/support/contact
PrivacyUrl:   https://www.ivanti.com/company/legal/privacy-policy
PackageName: Velocity Console
PackageUrl: https://www.wavelink.com/Download-Velocity_enterprise-app-modernization-Software/
```

**Comment by upintheairsheep** [comment]
Screenshot please

**Comment by upintheairsheep** [comment]
Fix hash

File: manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Ivanti.VelocityConsole
+PackageVersion: 2.1.16.21039
+Installers:
+- InstallerUrl: https://www.wavelink.com/download/Downloads.aspx?DownloadFile=27577&returnUrl=/Download-Velocity_enterprise-app-modernization-Software/
+  InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerSha256: D2E56B2349E6F8C21ABAE39E15F2F4B3190CA8B40910C484CED48E28E22076DD
```
Comment (stephengillie) [inline manifests/i/Ivanti/VelocityConsole/2.1.16.21039/Ivanti.VelocityConsole.installer.yaml:11]:
```suggestion
  InstallerSha256: 998FCAFB3C0806F5F49C7939E6162EF8E3892D0C61CD32131663F3CF35071F2B
```

**Comment by ItzLevvie** [comment]
[Policy] Area-External

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by stephengillie** [comment]
Closing PR as stale.

**Pull request closed without merging**